src/main/kotlin/no/nav/familie/ba/sak/kjerne/vilkårsvurdering/preutfylling/PreutfyllBosattIRiketService.kt

```diff
@@ -116,8 +124,10 @@ class PreutfyllBosattIRiketService(
         adresserForPerson: Adresser,
         behandling: Behandling,
     ): Set<VilkårResultat> {
+        val personopplysningGrunnlag = persongrunnlagService.hentAktivThrows(behandling.id)
+        persongrunnlagService.oppdaterStatsborgerskapPåPerson(personResultat.aktør.tilPerson(personopplysningGrunnlag))
         val erBosattINorgeTidslinje = lagErBosattINorgeTidslinje(adresserForPerson, personResultat)
-        val erNordiskStatsborgerTidslinje = pdlRestKlient.lagErNordiskStatsborgerTidslinje(personResultat)
+        val erNordiskStatsborgerTidslinje = lagErNordiskStatsborgerTidslinje(personResultat.aktør.tilPerson(personopplysningGrunnlag).statsborgerskap)
```

Comment:
Her bør `personopplysningGrunnlag` sendes inn som parameter, så slipper man å hente det ut på nytt. I tillegg blir koden litt lettere å lese hvis `person` er en egen variabel.
```suggestion
        val person = personResultat.aktør.tilPerson(personopplysningGrunnlag)
        persongrunnlagService.oppdaterStatsborgerskapPåPerson(person)
        val erBosattINorgeTidslinje = lagErBosattINorgeTidslinje(adresserForPerson, personResultat)
        val erNordiskStatsborgerTidslinje = lagErNordiskStatsborgerTidslinje(person).statsborgerskap)
```